src/App.css

```
@@ -329,114 +336,797 @@
+  min-height: 28px;
+  flex: 0 0 auto;
+  align-items: center;
+}
+
+.release-artist-chip-role-face {
+  display: inline-flex;
+  min-height: 26px;
+  align-items: center;
+  gap: 7px;
+  border-color: #bdc9b9;
+  border: 1px solid #bdc9b9;
```

Stylelint (17.10.0)

[error] Overridden property "border-color" by shorthand "border" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)